spec/test-site/index.html

```diff
@@ -9,7 +9,7 @@
 </head>
 <body>
     <div id="output"></div>
-    <script src="renderjson.js" type="text/javascript" charset="utf-8"></script>
+    <script src="{{ "renderjson.js" | absolute_url }}" type="text/javascript" charset="utf-8"></script>
```

Comment:
Added this to ensure the internals that use `url` and `baseurl` don't blow up when either is a Proc.